lib/heap/index.js

```diff
@@ -12,20 +12,19 @@ var alias = require('alias');
 
 var Heap = module.exports = integration('Heap')
   .global('heap')
-  .global('_heapid')
   .option('appId', '')
-  .tag('<script src="//d36lvucg9kzous.cloudfront.net">');
+  .tag('<script src="//cdn.heapanalytics.com/js/heap-{{ appId }}.js">');
 
 /**
  * Initialize.
  *
- * https://heapanalytics.com/docs#installWeb
+ * https://heapanalytics.com/docs/installation#web
  *
  * @param {Object} page
  */
 
 Heap.prototype.initialize = function(page){
-  window.heap=window.heap||[];window.heap.load=function(a){window._heapid=a;var d=function(a){return function(){window.heap.push([a].concat(Array.prototype.slice.call(arguments,0)));};},e=["identify","track"];for (var f=0;f<e.length;f++)window.heap[e[f]]=d(e[f]);};
+  window.heap=window.heap||[],window.heap.load=function(t,e){window.heap.appid=t,window.heap.config=e;for (var o=function(t){return function(){heap.push([t].concat(Array.prototype.slice.call(arguments,0)))}},p=["clearEventProperties","identify","setEventProperties","track","unsetEventProperty"],c=0;c<p.length;c++)heap[p[c]]=o(p[c])};
```

Comment:
I don't think this code needs to be minified (even though that's what we had earlier, copy pasta!).